website/docs/reference/commands.md

```diff
@@ -201,25 +201,8 @@ Accepts option groups: [coursier](./cli-options.md#coursier-options), [debug](./
 
 Compile and run Scala code.
 
-Specific run configurations can be specified with both command line options and using directives defined in sources.
-Command line options always take priority over using directives when a clash occurs, allowing to override configurations defined in sources.
-Using directives can be defined in all supported input source file types.
-
-For a run to be successful, a main method must be present on the classpath.
-.sc scripts are an exception, as a main class is provided in their wrapper.
-
-Multiple inputs can be passed at once.
-Paths to directories, URLs and supported file types are accepted as inputs.
-Accepted file extensions: .scala, .sc, .java, .jar, .md, .jar, .c, .h, .zip
-For piped inputs use the corresponding alias: _.scala, _.java, _.sc, _.md
-All supported types of inputs can be mixed with each other.
-
-To pass arguments to the actual application, just add them after `--`, like:
-
-```sh
-scala-cli run Main.scala AnotherSource.scala -- first-arg second-arg
-```
-
+You are currently viewing the basic help for the run sub-command. You can view the full help by running: 
+   [1mscala-cli run --help-full[0m
```

Comment:
It is not formatted correctly.

Reply:
It actually is formatted correctly, but not for the website...
I have mixed feelings about continuing to use console `--help` messages for the website reference docs.
`[1m` and `[0m` are incorrectly decoded console characters for `Console.BOLD` and `Console.RESET`, respectively.
I temporarily fixed it by removing those characters from the help message, but we should probably address this separately, later